### DISCUSSION TITLE
Welcome to godot-playfab Discussions!

### DISCUSSION THREAD

**Discussion by Structed** [Structed/godot-playfab#98] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  *Propose features!

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Herbcan** [comment]
hi, i can't find PlayFab Client Config in user://playfab_client_config.tres after setting autoload PlayFabManager, can i create a new file for it? or how to generate it by my own?

**Reply by Structed** [reply]
Hey, sorry, I didn't see this until now! I only got a notification about it today 🙁
This should get auto-generated after login.
If you are seeing a log message, that's just an information.
If there is indeed an error with stack trace, please do let me know and post an issue!
